.github/workflows/ci.yml

```diff
@@ -0,0 +1,242 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - master
+            - '[0-9]+.[0-9]+'
+
+    pull_request: ~
+
+jobs:
+    coverage:
+        name: Coverage
+        runs-on: ubuntu-latest
+        steps:
+            - name: Setup PHP
+              uses: shivammathur/setup-php@master
+              with:
+                  php-version: 7.3
+                  extension-csv: intl
+                  coverage: pcov
+
+            - name: Checkout
+              uses: actions/checkout@v1
+              with:
+                  fetch-depth: 50
+
+            - name: Install the dependencies
+              run: |
+                  composer global require symfony/flex
+                  composer install --no-interaction --no-suggest
+
+            - name: Generate the coverage report
+              run: php -d pcov.enabled=1 vendor/bin/phpunit --testsuite=coverage --coverage-clover=clover.xml --colors=always
+
+            - name: Upload the coverage report
+              run: bash <(curl -s https://codecov.io/bash) -f clover.xml
+              env:
+                  CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+
+    coding-style:
+        name: Coding Style
+        runs-on: ubuntu-latest
+        steps:
+            - name: Setup PHP
+              uses: shivammathur/setup-php@master
+              with:
+                  php-version: 7.3
+                  extension-csv: intl
+                  coverage: none
+
+            - name: Checkout
+              uses: actions/checkout@v1
+              with:
+                  fetch-depth: 50
+
+            - name: Install the dependencies
+              run: |
+                  composer global require symfony/flex
+                  composer install --no-interaction --no-suggest
+
+            - name: Check the coding style
+              run: |
+                  PHP_CS_CONFIG=default vendor/bin/php-cs-fixer fix --diff --diff-format=udiff --dry-run
+                  PHP_CS_CONFIG=legacy vendor/bin/php-cs-fixer fix --diff --diff-format=udiff --dry-run
+                  PHP_CS_CONFIG=template vendor/bin/php-cs-fixer fix --diff --diff-format=udiff --dry-run
+
+            - name: Analyze the code
+              run: vendor/bin/phpstan analyse core-bundle/src core-bundle/tests --level=3 --no-progress
+
+    tests:
+        name: PHP ${{ matrix.php }}
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                php: [7.2, 7.3]
```

Comment:
Requires 7.4 now, we're 2 weeks before GA :)

Reply:
I am going to create a separate PR for this, because currently the PHP 7.4 build fails because of the GD extension not being installable.